apps/bleskomat-server/data/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+export BLESKOMAT_SERVER_ADMIN_PASSWORD=$(npm run --silent generate:adminPasswordHash -- "$APP_PASSWORD" | xargs)
```

Comment:
Why use `xargs` here?

Reply:
Also doesn't look like `$APP_PASSWORD` as actually being passed in from the compose file.

Reply:
What about supporting this directly in your app? If your app accepted `BLESKOMAT_SERVER_ADMIN_PASSWORD_PLAINTEXT` and hashed it internally you wouldn't need the bash entrypoint wrapper, could handle errors safer (like empty password value) and it also wouldn't leak the plaintext password to other processes in the CLI flag for the `npm run` process.

This doesn't need to be a blocker for shipping on Umbrel, it's also ok how it's implemetned currently, but just a suggestion if you're open to changing it.

Reply:
> Why use `xargs` here?

xargs is used to trim white-space from around the password hash.

> Also doesn't look like $APP_PASSWORD as actually being passed in from the compose file.

I saw it used in other apps in Umbel, but didn't realize it was necessary to proxy its value via the docker compose file.

> What about supporting this directly in your app? If your app accepted BLESKOMAT_SERVER_ADMIN_PASSWORD_PLAINTEXT and hashed it internally you wouldn't need the bash entrypoint wrapper, could handle errors safer (like empty password value) and it also wouldn't leak the plaintext password to other processes in the CLI flag for the npm run process.

This sounds reasonable. I will add the new environment variable to the bleskomat-server project.